sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -107,6 +109,7 @@ case class CatalogTablePartition(
     if (parameters.nonEmpty) {
       map.put("Partition Parameters", s"{${parameters.map(p => p._1 + "=" + p._2).mkString(", ")}}")
     }
+    stats.foreach(s => map.put("Partition Statistics", s.simpleString))
```

Comment:
This needs a test case. Could you add a new test file to [this test suite](https://github.com/apache/spark/tree/4ecc648ad713f9d618adf0406b5d39981779059d/sql/core/src/test/resources/sql-tests/inputs) for analyzing and desc table partitions? You can generate the result file by running the command:

> SPARK_GENERATE_GOLDEN_FILES=1 build/sbt "sql/test-only *SQLQueryTestSuite"